build_tools/github_actions/runner/chown_workdir.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+set -euo pipefail
+
+# clean up any messes docker has left
+sudo chown -R runner:runner /home/runner
```

Comment:
Seems like this comment should go in `cleanup_workdir.sh`, not `chown_workdir.sh`? Maybe add a new comment in this file explaining what the `chown` is accomplishing?